pkg/boot/grub/config_test.go

```diff
@@ -12,8 +12,47 @@ import (
 	"testing"
 
 	"github.com/u-root/u-root/pkg/boot/boottest"
+	"github.com/u-root/u-root/pkg/mount"
+	"github.com/u-root/u-root/pkg/mount/block"
 )
 
+// fakeDevices returns a list of fake block devices and a pool of mount points.
+// The pool is pre-populated so that Mount is never called.
+func fakeDevices() (block.BlockDevices, *mount.Pool, error) {
+	// For some reason, Glob("testdata_new/*/") does not work here.
+	files, err := ioutil.ReadDir("testdata_new")
```

Comment:
likely because of the trailing / glob failed? Seems like it might be a bug?

Reply:
Not sure, but it's easy to reproduce:

```
echo 'package main; import "path/filepath"; import "fmt"; func main() { fmt.Println(filepath.Glob("/*/")); }' > main.go && go run main.go
```
vs

```
echo 'package main; import "path/filepath"; import "fmt"; func main() { fmt.Println(filepath.Glob("/*")); }' > main.go && go run main.go
```

The first prints nothing, while the second prints all files and directories. There's no way to Glob only directories.